jekyll.gemspec

```
@@ -38,7 +38,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency("i18n",                  "~> 1.0")
   s.add_runtime_dependency("jekyll-sass-converter", "~> 2.0")
   s.add_runtime_dependency("jekyll-watch",          "~> 2.0")
-  s.add_runtime_dependency("kramdown",              "~> 2.3")
+  s.add_runtime_dependency("kramdown",              "~> 2.3.1")
```

This locks kramdown to 2.3.x, rather than 2.x. I think you want something like this:

```suggestion
  s.add_runtime_dependency("kramdown",              "~> 2.3", ">= 2.3.1")
```